config/utils.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func getConfigYaml(path string) ([]byte, error) {
+	if strings.HasPrefix(path, "http") {
+		res, err := http.Get(path)
+		if err != nil {
+			return []byte{}, err
```

Comment:
`nil` is a valid value for `[]byte` type 🙂 You can use nil in the returns in case of error to avoid one allocation